lib/meta_tags/text_normalizer.rb

```diff
@@ -71,7 +71,8 @@ def self.helpers
     # @return [String] string with no HTML tags.
     #
     def self.strip_tags(string)
-      ERB::Util.html_escape helpers.strip_tags(string)
+      string ||= ''
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.